### DISCUSSION TITLE
December 2024 || Can't seem to get it to work

### DISCUSSION THREAD

**Discussion by moa20016** [balu100/dolby-vision-for-windows#11] [General]
I did this a while back, and it worked. The Dolby Vision citification in windows settings and pop up in content showed up. However, recently I see the certificate went away. So, I retried all the steps, and I can't get the certificate anymore nor does it show up on the Dolby Access. Am I the only one having this problem? or is it due to new Windows update?
![Screenshot 2024-12-25 175455](https://github.com/user-attachments/assets/483a7cd4-49df-45d8-afc9-ecde3b1c61bf)

**Comment by balu100** [comment]
Do the process over again, you can reset all to default with cru's reset-all.exe

**Reply by moa20016** [reply]
I have but I still get HDR certification not found. should I change my color profile in nvidia settings?

**Reply by moa20016** [reply]
alright now it shows up after a couple more tries
